app/scripts/controllers/landing.js

```diff
@@ -29,8 +29,9 @@ angular.module('confRegistrationWebApp')
     $scope.jumboImg = jumboImgs[_.random(0, jumboImgs.length - 1)];
 
     $scope.eventSearch = _.debounce(function(val) {
-      if(!val){ return; }
-      if(val.length < 2){
+      $scope.loadingSearchResults = true;
```

Comment:
If you move this to after the if statement that checks the length, you shouldn't have to deal with it in the if.